actioncable/lib/action_cable/route_wrapper.rb

```diff
@@ -0,0 +1,22 @@
+module ActionCable
+  module RouteWrapper
+    def internal_cable_path?
+      path =~ self.class.url_regex
+    end
+
+    def internal?
```

Comment:
Could/should AD offer a nicer way to do this, as we've now needed it more than once? A private `:internal` option on the route definition itself or something?